vignettes/basic-usage.Rmd

```diff
@@ -0,0 +1,160 @@
+
+## Usage: Packages
+
+```{r create_package_docs, eval = FALSE}
+create_package_docs(
+  product_name = "Fake Product", 
+  version = "vFake", 
+  specs = spec_df,                # a tibble containing stories and requirements to validate
+  release_notes_file = "NEWS.md", # file path to a formatted markdown doc of release notes.
+  auto_test_dir = "some_dir",     # directory containing automated test results
+  style_dir = "style_ref_dir"     # Directory that has style references for the generated docx files
+)
+```
+
+See [`?create_package_docs`](https://metrumresearchgroup.github.io/mrgvalidate/reference/create_package_docs.html) for other arguments to tweak how and where the documents are rendered.
+
+## Usage: Metworx Platform (experimental)
+
+```{r create_metworx_docs, eval = FALSE}
+create_metworx_docs(
+  product_name = "Fake Product", 
+  version = "vFake", 
+  specs = spec_df,                # a tibble containing stories and requirements to validate
+  release_notes_file = "NEWS.md", # file path to a formatted markdown doc of release notes.
+  auto_test_dir = "some_dir",     # directory containing automated test results
+  man_test_dir = "some_dir",      # directory containing automated test results
+  style_dir = "style_ref_dir"     # Directory that has style references for the generated docx files
+)
+```
+
+See [`?create_metworx_docs`](https://metrumresearchgroup.github.io/mrgvalidate/reference/create_metworx_docs.html) for other arguments to tweak how and where the documents are rendered.
+
+# Example
+
+If the documents produced by `create_package_docs()` or `create_metworx_docs()` don't look right, or you need to debug for any reason, you can run the pieces separately. For the demonstration below, we will use the following constants:
+
+```{r vars}
+TEST_INPUTS_DIR <- system.file("test-inputs", package = "mrgvalidate")
+output_dir <- file.path("mrgvalidate-create-validation-docs")
+product_name <- "example"
+
+# Filter to automatic tests only
+specs <- readRDS(file.path(TEST_INPUTS_DIR, "specs.RDS")) %>% dplyr::filter(!grepl("MAN", TestIds))
+```
+
+```{r setup dir, echo=FALSE}
+if (fs::dir_exists(output_dir)) fs::dir_delete(output_dir)
+fs::dir_create(output_dir)
+```
+
+```{r example1, echo=TRUE}
+res_df <- 
+  mrgvalidate::create_package_docs(
+  product_name = product_name,
+  version = "vFAKE",
+  repo_url = "git@github.com:metrumresearchgroup/mrgvalidate.git",
+  specs = specs,
+  release_notes_file = file.path(TEST_INPUTS_DIR, "release_notes_sample.md"),
+  auto_test_dir = file.path(TEST_INPUTS_DIR, "validation-results-sample"),
+  output_dir = output_dir
+)
+```
+
+```{r list_files, echo=TRUE}
+list.files(output_dir)
+```
+
+
+## Debugging
+Packages do not have manual tests, but for demonstration purposes lets include the previously filtered out. By default any missing *required* links will cause the docs to not be rendered. Additionally, a message displaying the missing links will be returned.
```

Comment:
Missing word at end of first sentence: "let's include the previously filtered out."